### DISCUSSION TITLE
Conflict when installing packages

### DISCUSSION THREAD

**Discussion by urthling** [airsdk/apm#114] [Q&A]
When installing Firebase, seeing the following -- does this mean that this needs to be resolved, or I can simply continue?

```
apm install com.distriqt.Firebase
✓ Found package: com.distriqt.Firebase@7.0.1    
✓ Found package: com.distriqt.playservices.Base@18.0.1   
✓ Found package: androidx.core@1.6.1   
✓ Found package: com.google.firebase.core@29.0.401 
✓ Found package: com.distriqt.playservices.AdsIdentifier@18.0.1   
✓ Found package: com.google.android.datatransport@3.0.0   
✓ Found package: com.distriqt.playservices.CloudMessaging@17.0.2   
CONFLICT :: fatal error : found [1] conflicting packages
CONFLICT :: com.distriqt.playservices.Base
CONFLICT :: ├── 18.0.1 required by: com.distriqt.Firebase@7.0.1
CONFLICT :: └── 17.5.1 required by: com.distriqt.Location@4.3.1
install :: 

```

**Comment by marchbold** [comment]
This is not an error but working correctly. 

Location and Facebook are the last of our extensions to be updated to the latest androidx/play services so this error is correctly informing you location extension cannot be used currently with the latest Firebase extension.

Until we update the location extension you will need to either remove it or use the previous version of all our extensions. 

To identify the previous versions you can do:

```
apm view com.distriqt.Firebase
```

Which will output 

```
com.distriqt.Firebase@7.0.1   Firebase core functionality including analytics
license
└── [private] airnativeextensions :: https://airnativeextensions.com/license
└── more info: https://airnativeextensions.com/extension/com.distriqt.Firebase
tags
└── [ ios android firebase  ]
versions
├──7.0.1 : 2022-03-15T00:00:00.000Z
├──7.0.0 : 2022-03-10T00:00:00.000Z
├──6.0.9 : 2021-11-11T00:00:00.000Z
└──6.0.8 : 2021-10-22T00:00:00.000Z
```

Then install the older version:

```
apm install com.distriqt.Firebase 6.0.9
```

Otherwise you can wait until we have updated the location extension. This one is requiring a lot of work due to changes in permissions. We are currently planning a release towards the end of this month / early next.
